test/verify/check-system-info

```diff
@@ -380,5 +380,43 @@ class TestSystemInfo(MachineCase):
         b.enter_page("/system")
 
 
+    @skipImage("PCP config not yet in manifest and no packagekit", "rhel-7-5")
```

Comment:
This is misleading - rhel-7-5 does have PackageKit. It's just that the old cockpit-system on rhel-7-5 does not have the install-on-demand feature.

Reply:
Changed, thanks.